docs/lazy-definitions.md

```diff
@@ -19,34 +19,44 @@ main = (Nat.add (S (S (S Z))) (S Z))
 
 Because of that, is recommended to use [supercombinator](https://en.wikipedia.org/wiki/Supercombinator) formulation to make terms be unrolled lazily, preventing infinite expansion in recursive function bodies.
```

Comment:
```suggestion
Because of that, it's recommended to use a [supercombinator](https://en.wikipedia.org/wiki/Supercombinator) formulation to make terms be unrolled lazily, preventing infinite expansion in recursive function bodies.
```